File: src/main/java/com/cle/onlinetestsystem/pojo/Teacher.java
package com.cle.onlinetestsystem.pojo;

import com.alibaba.excel.annotation.ExcelProperty;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.Data;

@Data
public class Teacher {
  @TableId
  private Long teacherId;//教师id
  @ExcelProperty(value = "教工号",index = 0)
  private String teacherNo;//教工号
  private String password;//密码
  @ExcelProperty(value = "教师姓名",index = 1)
  private String teacherName;//教师姓名
  private Integer isAuth;//是否管理
}

File: src/utils/Student/Teacher.ts
export default class Teacher {
  public teacherId: string
  public teacherNo: string
  public teacherName: string
  public isAuth: boolean
  constructor(
    teacherId: string,
    teacherNo: string,
    teacherName: string,
    isAuth: boolean = false
  ) {
    this.teacherId = teacherId
    this.teacherNo = teacherNo
    this.teacherName = teacherName
    this.isAuth = isAuth
  }
}
